test/functional_test.js

```diff
@@ -786,25 +781,23 @@ function runOnPrefix(apiPrefix) {
             'simplePushURL': pushURL
           }).expect(200).end(function(err) {
             if (err) throw err;
-            assert.calledOnce(statsdClient.count);
+            assert.calledOnce(statsdClient.increment);
             assert.calledWithExactly(
-              statsdClient.count,
-              "loop.activated-users",
-              1
-            );
+              statsdClient.increment,
+              "loop.activated-users");
```

Comment:
You mean:

    assert.calledWithExactly(
        statsdClient.increment, "loop.activated-users");

Because this doesn't: 

    assert.calledWithExactly(statsdClient.increment, "loop.activated-users");